spec/unit/etc_spec.rb

```diff
@@ -50,39 +51,85 @@
       end
     end
 
-    context "given an original Etc::Group struct with field values that cannot be converted to UTF-8" do
+    context "given an original Etc::Group struct with BINARY field values" do
+      original_encoding = Encoding.default_external
+
+      after(:each) do
+        Encoding.default_external = original_encoding
+      end
```

Comment:
We end up repeating this pattern a few times, and I wonder if it's going to come up more.

Might be worth putting this into a spec util helper function, something callable like
```
with_external_encoding(Encoding::EUC_KR) do
  let(:group) { Etc::Group.new }
  ...
```